src/highdicom/sr/templates.py

```diff
@@ -584,6 +892,92 @@ def __init__(
             )
             self.append(role_in_procedure_item)
 
+    @property
+    def name(self) -> str:
+        """str: name of the person"""
+        return self[0].value
+
+    @property
+    def login_name(self) -> Union[str, None]:
```

Comment:
I personally prefer `Union[str, None]` over `Optional[str]` for return values. I tend argue that there is no "optional" return value, since a function always returns a value. That value may be `None`.
In addition, the `Union[..., None]` style more naturally extends to more than two options (e.g., `Union[int, float, None]`). Unless you have a strong opinion about this, I would keep the current style.

Reply:
Ok, don't necessarily agree but we'll go with `Union` like we did in #74